### PR TITLE
AP-309 Check for subscription on Dashboard

### PR DESCRIPTION
**Pull request by jayeshsolanki93**

### CODE REVIEW

File: app/src/lumen/java/com/cliqz/browser/utils/ViewExtensions.kt [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+fun View.forEachChildView(closure: (View) -> Unit) {
+    closure(this)
+    val viewGroup = this as? ViewGroup ?: return
+    for (i in 0 until viewGroup.childCount) {
+        viewGroup.getChildAt(i).forEachChildView(closure)
+    }
+}
+
+fun View.enableView(isEnabled: Boolean) {
```
Comment (spacifici) [inline app/src/lumen/java/com/cliqz/browser/utils/ViewExtensions.kt]: please, rename this to `enableViewHierarchy`

File: app/src/lumen/res/layout/forget_tab.xml [hunk 2]
```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (spacifici) [inline app/src/lumen/res/layout/forget_tab.xml]: Please, do not add `RelativeLayout`s

Comment (jayeshsolanki93) [reply app/src/lumen/res/layout/forget_tab.xml]: @spacifici I copied it from the cliqz variant. Should I change the original layout file as well?

Comment (jayeshsolanki93) [reply app/src/lumen/res/layout/forget_tab.xml]: I refactored both the layouts to use `LinearLayout` as the root.

**Pull request merged**